fastlane/actions/gh_pages.rb

```diff
@@ -27,7 +39,7 @@ def self.run(params)
         # Download theme and generate docs
         UI.user_error!("svn not installed") unless system("command -v svn")
         sh "svn export https://github.com/rakutentech/ios-buildconfig/trunk/jazzy_themes jazzy_themes --force"
-        sh "bundle exec jazzy --output artifacts/docs/#{module_version} --theme jazzy_themes/apple_versions"
+        sh "bundle exec jazzy --output artifacts/docs/#{module_version} --theme jazzy_themes/apple_versions --exclude=RakutenWebClientKit"
```

Comment:
I think the exclusions should be configurable and passed in

Reply:
Oops--shouldn't be in this PR